CONTRIBUTING.md

```diff
@@ -77,6 +77,31 @@ Or run the documentation and browse to [http://localhost:1234/](http://localhost
 yarn start:docs
 ```
 
+### Component/Hook scaffolding
+If you are looking to contribute a brand new component or hook in a package that does not exist yet, please run the following command.
+```bash
+yarn plop
+```
+This will start a series of cli prompts to determine what template files and folders should be generated to help quick start your contribution.
+The prompts are as follows:
+```
+1. What type of project are you setting up?
+- Select "React Spectrum v3"
+
+2. scope name(s)
+- Select the package scopes that make sense for your contribution (e.g. react-spectrum if you are contributing a component, react-aria for an aria hook, etc).
+
+3. Package name, all lowercase (e.g. textfield)
+
+4. Component name, please use appropriate uppercase (e.g. TextField)
+- If you are contributing a non-component hook, just enter the name of your hook.
+
+5. Component css module name, blank if N/A. If unsure, check @adobe/spectrum-css-temp/components for a module containing the desired css (e.g. textfield)
+- If you cannot find the component css module name in @adobe/spectrum-css-temp/components, feel free to reach out to a team member on the git issue.
+```
```

Comment:
Maybe overkill? Maybe not clear enough? Open to opinions

Reply:
I beg to differ. I think it's important that this tutorial stands out for potential contributors to see, rather than being overlooked in just a few lines. I would suggest, though, to keep the steps outside a code block, maybe..?

Reply:
Sounds good, I've formatted it it with the proper indenting instead now